src/camera/pi/meowlpi/camera/camera.py

```diff
@@ -0,0 +1,40 @@
+from time import sleep
+from picamera import PiCamera
+from meowlpi.camera.streamer import MeowlPiStreamer
+import settings
```

Comment:
I follow this guide for sorting imports: https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement?answertab=votes#tab-top

Reply:
No, you use pylint.

Reply:
http://google.github.io/styleguide/pyguide.html#313-imports-formatting

Reply:
That's what I did.

Reply:
ok